taxi/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ('license_number', )
+    fieldsets = UserAdmin.fieldsets + (
+        ('Additional info', {'fields': ('license_number', )}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ('Additional info', {'fields': ('license_number', )}),
+    )
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ['manufacturer', 'model', ]
+    search_fields = ['model', ]
+    list_filter = ['model', ]
```

Comment:
You should filter by manufacturer
<img width="401" alt="image" src="https://github.com/user-attachments/assets/e63794ee-5392-45a8-99db-35b831c47e29">